libs/core/langchain_core/language_models/base.py

```diff
@@ -126,6 +130,10 @@ class BaseLanguageModel(
     )
     """Optional encoder to use for counting tokens."""
 
+    model_config = ConfigDict(
```

Comment:
is this new?

Reply:
Yes, it's required when using pydantic 2 since cache is an attribute on the chat model and the cache is not a base model